tests/test_rate_limiter.py

```
@@ -0,0 +1,62 @@
+import pytest
+import time
+from translate_srt import RateLimiter
+
+
+def test_rate_limiter_initial_state():
+    """Test initial state of rate limiter"""
+    limiter = RateLimiter(window_size=60, max_requests=3)
+    assert limiter.can_make_request() == True
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if limiter.can_make_request():` for truth checks

Replace with `limiter.can_make_request()`

(E712)